CONTRIBUTING.md

```diff
@@ -2,6 +2,33 @@
 
 We want to make contributing to this project as easy and transparent as possible.
 
+## Tests
+
+Ideally, unit tests should accompany every newly introduced exported function. We're always striving to increase the project's test coverage.
+
+If you need to mock an interface in your tests, follow the next steps:
+
+1. Comment the interface you'd like to mock following the example below
+
+    ```go
+    //go:generate mockery --name=fcmClient --output=. --case=underscore --inpackage
+    type fcmClient interface {
+        SendWithRetry(*fcm.Message, int) (*fcm.Response, error)
+    }
```

Comment:
```suggestion
    //go:generate mockery --name=nameOfClient --output=. --case=underscore --inpackage
    type nameOfClient interface {
    	...
    }
```

How do you feel about generalizing this example and referencing existing examples like fcm in an additional comment, like:

> For real-life implementation examples, check out existing services, for example [fcm](https://github.com/nikoksr/notify/blob/bda5705e4ee1cbf6b02bbb12679ed597334dee51/service/fcm/fcm.go#L27).

Reply:
Of course, it makes a lot more sense. Will do.